src/lib/util/compat.ml

```diff
@@ -153,3 +153,99 @@ module Type = struct
       match A.Id with B.Id -> Some Equal | _ -> None
   end
 end
+
+module In_channel = struct
+  open In_channel
```

Comment:
Ah yes unfortunately that one is not going to work since the module itself was added in 4.14; bummer.

Reply:
outch!